frontend/src/posapp/components/pos/flows/InvoiceManagement.vue

```
@@ -923,26 +991,60 @@ export default {
+		currentDraftSourceOption() {
+			return getDocumentSourceOption(this.currentDraftSource);
+		},
+		showDraftSourceSelector() {
+			return shouldShowDocumentSourceSelector(this.availableDraftSources);
+		},
+		canDeleteActiveDraftSource() {
+			return canDeleteDocumentSourceRecord(this.currentDraftSource);
+		},
+		draftHeaders() {
+			return [
+				{ title: __(this.currentDraftSourceOption.label), key: "name" },
```

ESLint

[error] '__' is not defined.

(no-undef)